### ISSUE TITLE
Could not find file/home/pi/node-sonos-http-api-settings.json

### ISSUE THREAD

**Issue by jaimesterz** [jishi/node-sonos-http-api#672]
Hi,

I'm new to all of this so please dumb it down as much as possible. I tried installing the API on a new raspberry pi. I installed it by cloning the git link and then used npm install --production. When I try to run npm start I receive the following error:

pi@raspberrypi:~/node-sonos-http-api $ npm start

> sonos-http-api@1.6.6 start /home/pi/node-sonos-http-api
> node server.js

2019-01-17T08:11:40.609Z INFO Could not find file /home/pi/node-sonos-http-api/settings.json
/home/pi/node-sonos-http-api/node_modules/music-metadata/lib/index.js:23
function parseStream(stream, mimeType, options = {}) {
                                               ^

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:414:25)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/pi/node-sonos-http-api/lib/helpers/file-duration.js:1:81)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)

npm ERR! Linux 4.14.79-v7+
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! sonos-http-api@1.6.6 start: `node server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sonos-http-api@1.6.6 start script 'node server.js'.
npm ERR! This is most likely a problem with the sonos-http-api package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node server.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls sonos-http-api
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/pi/node-sonos-http-api/npm-debug.log

**Comment by jishi** [comment]
Please pull the latest changes and run `npm install --production` again, and it should work. This was a known error that I hadn't fixed.
